pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -123,26 +126,54 @@ func (pc *PackageControllerClient) EnableCuratedPackages(ctx context.Context) er
 		values = append(values, "cronjob.suspend=true")
 	}
 
-	err := pc.chartInstaller.InstallChart(ctx, pc.chart.Name, ociURI, pc.chart.Tag(), pc.kubeConfig, "", values)
-	if err != nil {
+	if err := pc.chartInstaller.InstallChart(ctx, pc.chart.Name, ociURI, pc.chart.Tag(), pc.kubeConfig, "", values); err != nil {
 		return err
 	}
 
 	// Customers are currently requesting to show a warning in case
 	// credentials are not provided for curated packages
-	if err = pc.CreateCredentials(ctx); err != nil {
+	if err := pc.CreateAWSCredentials(ctx); err != nil {
 		logger.MarkWarning("  Unable to create credentials for curated packages: ", "warning", err)
 	}
 
```

Comment:
For the AWS creds are we *not* creating them if the env vars are not set? It seems like it would be easier for the customer if an empty secret was created and they could configure it later if they wanted to. I think that idea was shot down before, but I still hold that opinion.

Reply:
quotes are added to the aws-secret template.